### ISSUE TITLE
setting with dictionary does not seem to work for grid.linewidth?

### ISSUE THREAD

**Issue by michaelaye** [mwaskom/seaborn#427]
I'm going through the tutorial here: http://stanford.edu/%7Emwaskom/software/seaborn/tutorial/aesthetics.html
and am stuck at this line:
```python
sns.set_style('darkgrid', {'grid.linewidth': .5, 'axes.facecolor': '.8'})
sinplot()
```
My playing with the grid.linewidth parameter does not seem to do anything, plus the parameter is not being listed in the output of axes_style() afterwards, while the value of `axes.facecolor` can be changed and is being listed:
```python
{'axes.axisbelow': True,
 'axes.edgecolor': 'white',
 'axes.facecolor': '.8',
 'axes.grid': True,
 'axes.labelcolor': '.15',
 'axes.linewidth': 0.0,
 'font.family': ['sans-serif'],
 'font.sans-serif': ['Arial',
  'Liberation Sans',
  'Bitstream Vera Sans',
  'sans-serif'],
 'grid.color': 'white',
 'grid.linestyle': '-',
 'image.cmap': 'Greys',
 'legend.frameon': False,
 'legend.numpoints': 1,
 'legend.scatterpoints': 1,
 'lines.solid_capstyle': 'round',
 'text.color': '.15',
 'xtick.color': '.15',
 'xtick.direction': 'out',
 'xtick.major.size': 0.0,
 'xtick.minor.size': 0.0,
 'ytick.color': '.15',
 'ytick.direction': 'out',
 'ytick.major.size': 0.0,
 'ytick.minor.size': 0.0}
```

System:
* IPython master (today)
* Seaborn master (today)
* Safari 7.1.2 on OSX 10.9.5

**Comment by mwaskom** [comment]
Ah yeah, looks like that parameter has been moved to the `set_context`/`plotting_context` group of params, nice catch.

**Comment by mwaskom** [comment]
Closed with #566
